### ISSUE TITLE
Unhandled Runtime Error when upgrading to 5.1.0 from 5.0.0

### ISSUE THREAD

**Issue by lipis** [prisma/prisma#20480]
### Bug description

I just upgraded to 5.1.0 and getting an `Unhandled Runtime Error`, don't really know how to debug. `Category` is one of my enum tables in schema.

```
Unhandled Runtime Error
TypeError: Cannot set properties of undefined (setting 'Category')

Call Stack
eval
node_modules/.prisma/client/index-browser.js (430:42)
./node_modules/.prisma/client/index-browser.js
file:///Users/lipis/work/lim-expenses/.next/static/chunks/pages/index.js (17:1)
options.factory
/_next/static/chunks/webpack.js (707:31)
__webpack_require__
file:///Users/lipis/work/lim-expenses/.next/static/chunks/webpack.js (37:33)
fn
/_next/static/chunks/webpack.js (362:21)
eval
node_modules/@prisma/client/index-browser.js (1:15)
./node_modules/@prisma/client/index-browser.js
file:///Users/lipis/work/lim-expenses/.next/static/chunks/pages/index.js (27:1)
options.factory
/_next/static/chunks/webpack.js (707:31)
__webpack_require__
file:///Users/lipis/work/lim-expenses/.next/static/chunks/webpack.js (37:33)
fn
/_next/static/chunks/webpack.js (362:21)
eval
webpack-internal:///./lib/date-utils.ts (27:72)
./lib/date-utils.ts
file:///Users/lipis/work/lim-expenses/.next/static/chunks/pages/index.js (9363:1)
options.factory
/_next/static/chunks/webpack.js (707:31)
__webpack_require__
file:///Users/lipis/work/lim-expenses/.next/static/chunks/webpack.js (37:33)
fn
/_next/static/chunks/webpack.js (362:21)
eval
webpack-internal:///./components/branches/BranchItem.tsx (4:73)
./components/branches/BranchItem.tsx
file:///Users/lipis/work/lim-expenses/.next/static/chunks/pages/index.js (9198:1)
options.factory
/_next/static/chunks/webpack.js (707:31)
__webpack_require__
file:///Users/lipis/work/lim-expenses/.next/static/chunks/webpack.js (37:33)
fn
/_next/static/chunks/webpack.js (362:21)
eval
webpack-internal:///./pages/index.tsx (8:89)
./pages/index.tsx
file:///Users/lipis/work/lim-expenses/.next/static/chunks/pages/index.js (9493:1)
options.factory
/_next/static/chunks/webpack.js (707:31)
__webpack_require__
file:///Users/lipis/work/lim-expenses/.next/static/chunks/webpack.js (37:33)
fn
/_next/static/chunks/webpack.js (362:21)
eval
node_modules/next/dist/build/webpack/loaders/next-client-pages-loader.js?absolutePagePath=%2FUsers%2Flipis%2Fwork%2Flim-expenses%2Fpages%2Findex.tsx&page=%2F! (5:15)
execute
node_modules/next/dist/client/route-loader.js (233:50)
```

### Prisma information

```prisma
generator client {
  provider = "prisma-client-js"
}

generator erd {
  provider  = "prisma-erd-generator"
  output    = "../prisma/ERD.svg"
  tableOnly = true
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

enum Method {
  bank
  cash
  check
  credit
  delivery
}

enum Category {
  expense
  recurrent
  report
  revenue
  salary
  share
  vendor
}

enum Duration {
  day
  week
  month
  quarter
  semester
  year
  all
}

model Config {
  id           Int    @id @default(0)
  trialDays    Int    @default(60)
  priceMonth   Int    @default(29)
  priceYear    Int    @default(299)
  priceTax     Float  @default(0)
}

...

```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: macOS
- Database: PostgreSQL
- Node.js version: v18.17.0

### Prisma Version

```
5.1.0
```

**Comment by GuysmoB** [comment]
Same here

**Comment by jharrell** [comment]
@lipis thanks for the report! We've confirmed that this is an issue in `5.1.0` and are working on a patch release to resolve. We will update you here!

**Comment by SevInf** [comment]
Hi @lipis and @GuysmoB.
The fix will be a part of a patch release. You can try out integration version `5.2.0-integration-fix-enums-browser.1` meanwhile to verify that fix works for you. Normally, we don't recommend using integration version in production, however, right now this is just 5.1 with this issue fixed and it's very likely it will become 5.1.1. 

Thank you for reporting the issue again!

**Comment by ahkhanjani** [comment]
> The fix will be a part of a patch release. You can try out integration version `5.2.0-integration-fix-enums-browser.1` meanwhile to verify that fix works for you.

Seems to have fixed the problem.

**Comment by sunnysingh** [comment]
I have the same issue and version `5.2.0-integration-fix-enums-browser.1` fixes it.

**Comment by TomLampard** [comment]
sorry everyone, but how do I install  `5.2.0-integration-fix-enums-browser.1` ? Just a npm i --save-dev prisma@5.2.0-integr etc etc?

**Comment by TomLampard** [comment]
I'm having the same issue, I tried moving back to Prisma 5.0.0 and project still bugged.  How can I install this 5.2.0 integration?

**Comment by janpio** [comment]
```
npm install -D prisma@5.2.0-integration-fix-enums-browser.1
npm install @prisma/client@5.2.0-integration-fix-enums-browser.1
npx prisma generate
```
That should do the trick @TomLampard.

**Comment by TomLampard** [comment]
@janpio  Thanks kindly

**Comment by elysiantech** [comment]
@janpio Thanks, solved my problems!!building from docker" => [build  9/11] RUN npx prisma generate                                                                        16.8s
 => => # npm WARN exec The following package was not found and will be installed: prisma@5.1.0"

**Comment by SevInf** [comment]
Fix released in 5.1.1
